backend/tracim_backend/lib/core/user.py

```diff
@@ -233,7 +232,7 @@ def user_with_email_exists(self, email: str) -> bool:
             self.get_one_by_email(email)
             return True
         # TODO - G.M - 09-04-2018 - Better exception
```

Comment:
Can you open a technical debt issue for this ?

Reply:
done #1635